src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactGridLayout from 'react-grid-layout';
 import TransitComponent from '@databraid/transit-widget/lib';
-import SlackComponent from '@databraid/slack-widget/lib';
+import SlackComponent from '@databraid/slack-widget/lib/App';
```

Comment:
Is this all there is with regards to imports from the slack widget? no reducers or API or nothing?